recipe/meta.yaml

```diff
@@ -30,11 +30,13 @@ requirements:
     - {{native}}toolchain  # [win]
     - gcc                  # [not win]
     - curl
+    - libcurl
```

Comment:
conda-forge `libcurl` is named `curl`. `libcurl` is a defaults package. I am unsure if that is an alias to `curl` or something else.

Reply:
Huh-- my bad. Installing just curl doesn't work, but libcurl does:

```
conda create -n rcurl-test
source activate rcurl-test
conda install -c conda-forge r-rcurl
(rcurl-test) -bash-4.1$ Rscript -e 'library(RCurl')
-bash: syntax error near unexpected token `)'
(rcurl-test) -bash-4.1$ Rscript -e 'library(RCurl)'
Loading required package: methods
Loading required package: bitops
Error: package or namespace load failed for âRCurlâ in dyn.load(file, DLLpath = DLLpath, ...):
 unable to load shared object '/gpfs/users/kbdt501/local/share/bcbio/anaconda/envs/rcurl-test/lib/R/library/RCurl/libs/RCurl.so':
  /gpfs/users/kbdt501/local/share/bcbio/anaconda/envs/rcurl-test/lib/R/library/RCurl/libs/../../../../libcurl.so.4: undefined symbol: SSL_CTX_set_alpn_protos
Execution halted
```

```
conda install -c conda-forge libcurl
(rcurl-test) -bash-4.1$ Rscript -e 'library(RCurl)'
Loading required package: methods
Loading required package: bitops
(rcurl-test) -bash-4.1$
```

Reply:
We need to investigate a few things here, like where is the `r-curl` you are using is coming from, is conda-forge's `r-curl` linking to `defaults` `libcurl` (if so we need to fix it), or if we have an issue with our `curl` that should be fixed. However, specifying `defaults` `libcurl` here is not the solution.

BTW, can you modify this PR to add a test for this issue instead of adding `libcurl`?

Reply:
I could note reproduce this on Linux. I wonder if this is related to https://github.com/conda-forge/curl-feedstock/issues/23

Reply:
Thanks, it works fine on CentOS and Ubuntu, but not the RedHat machine I tested it on. The RedHat machine is grabbing a different version of r-curl from conda-forge:

`CentOS: r-rcurl:         1.95_4.8-r3.4.1_0 conda-forge`
`CentOS: curl:            7.55.1-0          conda-forge`
`RedHat: r-curl:          3.1-r3.4.1_0     conda-forge`
`RedHat: curl:            7.55.1-0          conda-forge`

which might be where the issue is coming from.

Reply:
Aren't `r-rcurl` and `r-curl` different packages?

Reply:
Sorry, I had a typo when I tested, I updated the comment.

Reply:
My guess is that our `curl` is missing that symbol and the system `curl` from your CentOS works b/c it has that and RedHat does not. Either way we should fix that in our curl lib and not here.

Reply:
Thanks, it isn't all RedHats; I just tried a Vagrant box with RHEL7 and it worked fine. I'll close this p/r and open up an issue over there.

Reply:
Thanks. I'll try to follow up there but this week will be complicated for me. Hopefully someone else can chime in on that issue.